python/python/glide/async_commands/sorted_set.py

```diff
@@ -0,0 +1,150 @@
+# Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0
+
+from enum import Enum
+from typing import List, Optional, Union
+
+
+class InfBound(Enum):
+    """
+    Enumeration representing numeric and lexicographic positive and negative infinity bounds for sorted set.
+    """
+
+    POS_INF = {"score_arg": "+inf", "lex_arg": "+"}
```

Comment:
Add documentation to the enum values (even if it's short as 'positive infinity'), make sure to use the enum convention for docstring